NewDesignExample.cs

```diff
@@ -0,0 +1,59 @@
+﻿///////////////////////// COMMANDS CORE
```

Comment:
Do you need to check this in? Particularly at the src root?

Reply:
Unintentional thanks.  Has been removed in new commit to this branch.